engines/director/detection.cpp

```diff
@@ -29,163 +29,93 @@
-	{ "noir",				"Noir: A Shadowy Thriller"},
-	{ "operafatal",			"Opera Fatal"},
-	{ "paws",				"P.A.W.S.: Personal Automated Wagging System"},
-	{ "phantasplanet",		"Phantasmagoria Amusement Planet"},
-	{ "pitfall",			"Pitfall: The Mayan Adventure" },
-	{ "planetarizona",		"Escape from Planet Arizona" },
-	{ "prescue",			"Paradise Rescue" },
-	{ "refixion1",			"Refixion"},
-	{ "refixion2",			"Refixion II: Museum or Hospital"},
-	{ "refixion3",			"Refixion III: The Reindeer Story"},
-	{ "rodney",				"Rodney's Funscreen"},
-	{ "saillusion",     	"Scientific American Library: Illusion" },
-	{ "sakin2",				"Sakin II"},
-	{ "santafe1",			"Santa Fe Mysteries: The Elk Moon Murder"},
-	{ "sauniverse",     	"Scientific American Library: The Universe" },
-	{ "sciencesmart",		"Science Smart"},
-	{ "screamingmetal",		"Screaming Metal"},
-	{ "shanghai",			"Shanghai: Great Moments"},
-	{ "simpsons",			"The Simpsons: Cartoon Studio"},
-	{ "skyborg",			"SkyBorg: Into the Vortex"},
-	{ "snh",				"A Silly Noisy House"},
-	{ "splayer",			"The Simpsons: Cartoon Studio Player"},
-	{ "spyclub",			"Spy Club" },
-	{ "spycraft",			"Spycraft: The Great Game" },
-	{ "staytooned",			"Stay Tooned!" },
-	{ "superspy",			"SuperSpy 1" },
-	{ "teamxtreme1",		"Operation: Weather Disaster" },
-	{ "teamxtreme2",		"Operation: Eco-Nightmare" },
-	{ "teddybear",			"Operation Teddy Bear" },
-	{ "the7colors",			"The Seven Colors: Legend of PSY-S City"},
-	{ "totaldistortion",	"Total Distortion"},
-	{ "trekborg",			"Star Trek: Borg"},
-	{ "trekguideds9",		"Star Trek: Deep Space Nine Episode Guide"},
-	{ "trekguidetng",		"Star Trek: The Next Generation Episode Guide"},
-	{ "trekklingon",		"Star Trek: Klingon"},
-	{ "trekomni",			"Star Trek Omnipedia"},
-	{ "trekpedia98",		"Star Trek Encyclopedia 1998"},
-	{ "trektech",			"Star Trek: The Next Generation Interactive Technical Manual"},
-	{ "tri3dtrial",			"Tri-3D-Trial"},
-	{ "twistynight1",		"Twisty Night #1"},
-	{ "twistynight2",		"Twisty Night #2"},
-	{ "twistynight3",		"Twisty Night #3"},
-	{ "ushistory",			"The History of the United States for Young People"},
-	{ "vvcyber",			"Victor Vector & Yondo: The Cyberplasm Formula"},
-	{ "vvdinosaur",			"Victor Vector & Yondo: The Last Dinosaur Egg"},
-	{ "vvharp",				"Victor Vector & Yondo: The Hypnotic Harp"},
-	{ "vvvampire",			"Victor Vector & Yondo: The Vampire's Coffin"},
-	{ "warlock", 			"Spaceship Warlock"},
-	{ "wishbone", 			"Wishbone and the Amazing Odyssey"},
-	{ "wrath",				"Wrath of the Gods"},
-	{ "xanthus",			"Xanthus"},
-	{ "xfua",				"The X-Files Unrestricted Access"},
-	{ "ybr1",				"Yellow Brick Road"},
-	{ "ybr2",				"Yellow Brick Road II"},
-	{ "ybr3",				"Yellow Brick Road III"},
-	{ "znemesis",			"Zork Nemesis: The Forbidden Lands"},
-	{ 0, 0 }
-};
-
 #include "director/detection_tables.h"
 
-static const char *directoryGlobs[] = {
-	"install",
-	"l_zone",
-	"win_data",	// L-ZONE
-	0
-};
+namespace Director {
+	const char *directoryGlobs[] = {
+		"install",
```

Comment:
According to our code formatting guidelines, `namespace` does not increase the indentation. I'll fix it after the merge